client/lib/analytics/utils/jetpack-cart-to-ga-purchase.ts

```diff
@@ -0,0 +1,28 @@
+import { ResponseCart } from '@automattic/shopping-cart';
+import { GaItem, jetpackProductToGaItem } from './jetpack-product-to-ga-item';
+import { WpcomJetpackCartInfo } from './split-wpcom-jetpack-cart-info';
+
+export type GaPurchase = {
+	value: number;
+	currency: string;
+	tax?: number;
+	transaction_id: string;
+	coupon: string;
+	items: GaItem[];
+};
+
+export function jetpackCartToGaPurchase(
+	orderId: string,
+	cart: ResponseCart,
+	cartInfo: WpcomJetpackCartInfo
+): GaPurchase {
+	return {
+		transaction_id: orderId,
+		coupon: cart.coupon,
+		currency: 'USD', // we track all prices in USD
+		value: cartInfo.jetpackCost,
```

Comment:
Are we sure `value` is the amount in USD, at this point?

Reply:
Good catch, I was trying to think whether it's better to use localized currencies or stick to one - and in the end, I've decided to hold on to the way we do it currently (so to track everything in USD). but I forgot to change `cartInfo.jetpackCost` to `cartInfo.jetpackCostUSD`